notebooks/slicing_tutorial_WiC.ipynb

```diff
@@ -0,0 +1,864 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Snorkel-Workshop: Slicing Tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "To start, let's make sure that we have the right paths/environment variables set by following the instructions in `snorkel-superglue/README.md`.\n",
+    "\n",
+    "Specifically, ensure that:\n",
+    "* `snorkel` is installed\n",
+    "* `PYTHONPATH` includes root of `snorkel-superglue`\n",
+    "* `SUPERGLUEDATA` is set where [download_superglue_data.py](https://github.com/HazyResearch/snorkel-superglue/blob/staging/download_superglue_data.py) was called."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys, os\n",
+    "import pandas as pd\n",
+    "from pathlib import Path\n",
+    "\n",
+    "from snorkel.mtl.data import MultitaskDataLoader\n",
```

Comment:
Before all these load statements, I'd add a 1-2 sentences reminding why all the "mtl" stuff